portfolio/src/main/webapp/index.html

```diff
@@ -240,6 +202,70 @@ <h1>Projects</h1>
 
         </div>
 
+        <div class="hidden" id="chart-wrapper">
```

Comment:
You should give this chart some `box-shadow` so it appears to hover over content. A little goes a long way here, so something like `box-shadow: grey 0px 0px 1px 0px` should give it that 3D effect.